data/styles/default/include/Icons.mapcss

```diff
@@ -516,16 +516,18 @@ area|z16[historic=fort],
 node|z16[historic=castle],
 area|z16[historic=castle]
 {icon-image: remains-m.svg;text-offset: 1;}
-node|z17-[historic=ruins][name],
-area|z17-[historic=ruins][name],
-node|z17-[historic=fort],
-area|z17-[historic=fort],
-node|z17-[historic=castle],
-area|z17-[historic=castle]
+node|z16-[historic=fort],
+area|z16-[historic=fort],
+node|z16-[historic=castle],
+area|z16-[historic=castle]
 {icon-image: remains-m.svg;}
 
-node|z17-[historic=archaeological_site],
-area|z17-[historic=archaeological_site]
+node|z12-[historic=ruins],
+area|z12-[historic=ruins]
+{icon-image: remains-m.svg;}
+
+node|z12-[historic=archaeological_site],
+area|z12-[historic=archaeological_site]
 {icon-image: archaeological-site-m.svg;}
```

Comment:
Same here because archaeological_site is also a quite common tag.
Better to introduce subtypes ```archaeological_site=*```